cluster-scope/overlays/prod/moc/infra/ingresscontrollers/default-ingress-certificate.enc.yaml

```diff
@@ -1,95 +0,0 @@
-apiVersion: v1
```

Comment:
You've deleted this file, but it needs to be removed from `secret-generator.yaml`